src/pages/payments/register/form/Form.module.css

```
@@ -0,0 +1,8 @@
+.payments {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+
+  max-width: 600px;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)